File: csharp/KyberKem.cs
using System;
using System.Collections.Generic;
using System.Security.Cryptography;
using System.Text;

namespace CryptoCorpus
{
    public class KyberKem
    {
        public const string DefaultCipher = "aes-256-gcm";
        public const string DefaultHash = "SHA-256";

        public string Sha256Hex(string data)
        {
            using var sha = SHA256.Create();
            byte[] hash = sha.ComputeHash(Encoding.UTF8.GetBytes(data));
            var sb = new StringBuilder();
            foreach (byte b in hash) sb.Append(b.ToString("x2"));
            return sb.ToString();
        }

        public static void Run()
        {
            var algorithms = new List<string> { "SM3", "GOST", "Dilithium", "scrypt" };
            Console.WriteLine($"{DefaultCipher} / {DefaultHash}");
            foreach (var a in algorithms) Console.WriteLine("configured: " + a);
        }
    }


// almost compiles: intentional single-token defect (AES, SHA-256)

File: typescript/kyberKem.ts
// KyberKem -- synthetic crypto naming demo (AES / SHA-256 names).
import { createHash } from 'node:crypto';

const DEFAULT_CIPHER: string = 'AES_256_GCM';
const DEFAULT_HASH: string = 'SHA_256';

export class KyberKem {
  sha256Hex(data: string): string {
    return createHash('sha256').update(data).digest('hex');
  }
}

const algorithms: string[] = ['RC5', 'CRC32', 'IDEA', 'ECDSA'];
const svc = new KyberKem();
console.log(`${DEFAULT_CIPHER} / ${DEFAULT_HASH`);
algorithms.forEach((a: string): void => console.log('configured:', a));

// almost compiles: intentional single-token defect (AES, SHA-256)
